src/test/java/org/lisoft/lsml/model/item/BallisticWeaponTest.java

```diff
@@ -42,10 +42,10 @@ public void testCUAC10() throws Exception {
         final BallisticWeapon cut = (BallisticWeapon) ItemDB.lookup(1206);
 
         assertTrue(cut.getName().contains("C-ULTRA AC/10"));
-        assertEquals(3, cut.getAmmoPerPerShot());
+        assertEquals(3, cut.getRoundsPerShot());
         assertEquals(10.0, cut.getDamagePerShot(), 0.0001);
 
-        final double expectedSecondsPerShot = cut.getCoolDown(null) + 0.11 * 2;
```

Comment:
@sestrand says: when you pull the trigger during cooldown, there is no additional time added to the cooldown for that interim shot.  I went into the game and tested against my target self with a stopwatch .  I don't find any additional time... interim shots cant fire before .11 after the first shot, but the cooldown is just the cooldown from completion of the first shot.

Reply:
I'm not sure I follow? When you pull down the trigger, you fire 1 round immediately then 2 more rounds each with 0.11 s between them, and the cooldown starts. This is the raw firing period. If you pull the trigger during the cooldown, that's double fire and that only affects eFP, not rFP that's verified in this test (which is admittedly brittle, should use a fake weapon).

> I went into the game and tested against my target self with a stopwatch.
A human with a stop watch isn't accurate enough to measure this without a more thorough testing procedure (which you might have done, but you didn't describe it so I do not know how you tested).

To verify I took a TBR-PRIME that doesn't have any cooldown quirks that affect UACs, I made sure it has no mech skills and equipped 1x C-UAC10 and 207 rounds of ammo. I dropped into testing grounds. I then used a stop watch on my phone and pressed it simulatneously with the trigger button on my mouse that I then held down. I keept holding down the fire button until around 1 minute has passed (this is so that any timing inaccuracies will be divided down) and then I stop the stop watch when I see the next projectile fire after the ammo level has reached 135. In other words, when I stop the timer, the cooldown of the firing period that put us at 135 ammo has just expired so we measure a full firing period at the end. 

The time measured was: 1:05.57. We went from 207 to 135 rounds, results in (207 - 135)/3 = 24 fire periods. 
Which results in: 1:05.57 / 24 = 65.57 / 24 = 2.73 seconds per fire period.

However we must correct for the average 0.25 seconds reaction speed of humans to visual stimulus, making it:
1:05.32 / 24 = 65.32 / 24 = 2.72 seconds per fire period.

The game files for C-UAC10 have the following: Cooldown = 2.5, numFiring = 3, volleyDelay = 0.11.
Thus the raw firing period for this weapon, under the current LSML calculations (that are changed in this PR) would be: 2.5 + (3-1)*0.11 = 2.72. This matches exactly with the experimental result, therefore I must conclude that based on this experiment, the current LSML behaviour is correct and the change proposed here is incorrect.